server.py

```diff
@@ -1,23 +1,39 @@
 import socket
+import datetime
 
-ADDR = ('127.0.0.1', 7000)
-
-socket = socket.socket(
-    socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
-    )
-
-socket.bind(ADDR)
-socket.listen(1)
-
-while True:
-    try:
-        conn, addr = socket.accept()
-        while True:
-            msg = conn.recv(16)
-            print msg
-            conn.sendall(msg)
-            if len(msg) < 16:
-                conn.close()
-                break
-    except KeyboardInterrupt:
-        break
\ No newline at end of file
+
+def response_ok():
+    content_type = "text/plain"
+    content_length = "1354"
+    date = datetime.datetime.strftime(datetime.datetime.now(), '%Y-%m-%d %H:%M')
```

Comment:
This does not create a compliant date/time string.  Review the lecture materials for HTTP wherein it is described exactly how to get the right format.